### DISCUSSION TITLE
ECHIPA LAURIENII

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#106] [TEAMs]
Aici sunt membrii echipei LAURIENII

**Comment by sebiboga** [comment]
LAURENTIU BALUTA
role: scraper developer
discord: Laurentiu

**Comment by AdrianaCionca** [comment]
Adriana Cionca
role: Manual Tester
discord: Adriana Cionca

**Comment by AdrianaCionca** [comment]
Adina Boeru
role: Manual Tester
discord: ADINA BOERU

**Comment by AdrianaCionca** [comment]
Adrian Vanca
role: Manual Tester
discord: Adi Vanca

**Comment by AdrianaCionca** [comment]
Andrei Baldean
role: Manual Tester
discord: andrei.baldean

**Comment by sebiboga** [comment]
Bogdan Dumitrache
role: Manual Tester
Bogdan D
